packages/manager/apps/hycu/public/translations/hycu/dashboard/Messages_fr_FR.json

```
@@ -34,15 +35,17 @@
   "hycu_dashboard_regenerate_upload_confirm": "Regénérer",
   "hycu_dashboard_upload_cancel": "Annuler",
   "hycu_dashboard_upload_confirm": "Activer",
-  "hycu_dashboard_upload_valid_file": "Merci de sélectionner un fichier valide",
+  "hycu_dashboard_upload_file_too_big": "La taille du fichier ne peut pas excéder 1Mo",
+  "hycu_dashboard_upload_file_bad_type": "Le fichier fourni n'est pas un fichier .req",
```

To easy any change, can you use variable instead (no further translation needed)
```suggestion
  "hycu_dashboard_upload_file_bad_type": "Le fichier fourni n'est pas un fichier {{extension}}",
```